schwifty/bank_registry/manual_it.json

```diff
@@ -1,666 +1,1122 @@
+		"primary": true,
+		"short_name": "BCC DEL TERRITORIO LOMBARDO"
+	},
+	{
+		"bank_code": "08736",
+		"bic": "ICRAITRRKG0",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO DI PONTASSIEVE - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BANCA DI CREDITO COOPERATIVO DI PONTASSIEVE"
+	},
+	{
+		"bank_code": "08765",
+		"bic": "ICRAITRRL90",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO DI RECANATI E COLMURANO - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BANCA DI CREDITO COOPERATIVO DI RECANATI E COLMURANO"
+	},
+	{
+		"bank_code": "08784",
+		"bic": "CCRTIT2TMPR",
+		"country_code": "IT",
+		"name": "BANCA MONTE PRUNO CREDITO COOPERATIVO DI FISCIANO, ROSCIGNO E LAURINO SC",
+		"primary": true,
+		"short_name": "BANCA MONTE PRUNO CREDITO COOPERATIVO DI FISCIANO, ROSCIGNO E LAURINO SC"
+	},
+	{
+		"bank_code": "08794",
+		"bic": "CCRTIT2TRMB",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO LAUDENSE - LODI SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BCC LAUDENSE - LODI SOCIETA' COOPERATIVA"
+	},
+	{
+		"bank_code": "08807",
+		"bic": "ICRAITRRBSG",
+		"country_code": "IT",
+		"name": "BCC DI VERONA E VICENZA CREDITO COOPERATIVO - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BCC DI VERONA E VICENZA CREDITO COOPERATIVO - SOCIETA' COOPERATIVA"
+	},
+	{
+		"bank_code": "08810",
+		"bic": "CCRTIT2TSGR",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO DI SAN GIOVANNI ROTONDO - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BANCA DI CREDITO COOPERATIVO DI SAN GIOVANNI ROTONDO"
+	},
+	{
+		"bank_code": "08851",
+		"bic": "ICRAITRRNN0",
+		"country_code": "IT",
+		"name": "TERRE ETRUSCHE DI VALDICHIANA E DI MAREMMA - CREDITO COOPERATIVO- S.C.",
+		"primary": true,
+		"short_name": "BANCA TEMA"
+	},
+	{
+		"bank_code": "08852",
+		"bic": "CCRTIT2TRMB",
+		"country_code": "IT",
+		"name": "ROMAGNABANCA CREDITO COOPERATIVOS.C.",
+		"primary": true,
+		"short_name": "ROMAGNABANCA CREDITO COOPERATIVO"
+	},
+	{
+		"bank_code": "08869",
+		"bic": "ICRAITRRO50",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO BERGAMO E VALLI - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BANCA DI CREDITO COOPERATIVO BERGAMO E VALLI"
+	},
+	{
+		"bank_code": "08871",
+		"bic": "CCRTIT2TSBE",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO DI SPELLO E DEL VELINO SOC. COOP.",
+		"primary": true,
+		"short_name": "BCC DI SPELLO E DEL VELINO"
+	},
+	{
+		"bank_code": "08883",
+		"bic": "CCRTIT2TBDB",
+		"country_code": "IT",
+		"name": "BANCA DI BOLOGNA CREDITO COOPERATIVO SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BANCA DI BOLOGNA"
+	},
+	{
+		"bank_code": "08899",
+		"bic": "ICRAITRRTR0",
+		"country_code": "IT",
+		"name": "CASSA RURALE - BANCA DI CREDITO COOPERATIVO DI TREVIGLIO - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "CASSA RURALE"
```

Comment:
should we keep ` BANCA DI CREDITO COOPERATIVO DI TREVIGLIO` to differentiate from the rest of `Cassa Rurale` branches?